.gitignore

```diff
@@ -2,6 +2,11 @@
 /phpunit.xml
 ###< phpunit/phpunit ###
 
+###> squizlabs/php_codesniffer ###
```

Comment:
Not actually added by Flex...

Reply:
Opened https://github.com/symfony/recipes-contrib/pull/456, which would have done it.